frontend/src/assets/projects.js

```diff
@@ -98,39 +98,39 @@ const projects = [
         documentation: ''
       }
     ],
-    website: ''
+    website: 'http://bankempatii.pl/'
   },
   {
-    description: '',
-    githubLink: '',
+    description: 'Oryginalny StreetMix jest dziełem CodeforAmerica, które w 2013 roku podczas hacknightu postanowiło stworzyć narzędzie, które pomoże miejskim planistom w przedstawianiu koncepcji ich planu na rewitalizacje miejskiej przestrzeni. Najczęstszym rozwiązaniem po jakie sięgają planiści jest wykonanie makiet z papieru, które zwizualizują krajobraz.  Tworząc internetową wersję tego działania, planiści mogą dotrzeć do szerszego grona odbiorców niż podczas samych spotkań, a także pozwolić członkom społeczności na dzielenie się i remiksowanie swoich dzieł. Jest to jeden z pierwszych naszych projektów, które mieliśmy przyjemność przetłumaczyć tak, aby można było wykorzystać te rozwiązani na polskim gruncie.',
```

Comment:
- Streetmix, not StreetMix
- Code for America - I think that's the correct name (with spaces).
- "rewlitalizacje" or "rewitalizację"
- double space before: "Tworząc internetową (...)"
- "rozwiązani" - missing "e"